dev/io.openliberty.microprofile.telemetry.1.0.internal_fat/fat/src/io/openliberty/microprofile/telemetry/internal_fat/TelemetryConfigEnvTest.java

```diff
@@ -0,0 +1,68 @@
+ * are made available under the terms of the Eclipse Public License 2.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-2.0/
+ * 
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package io.openliberty.microprofile.telemetry.internal_fat;
+
+import static com.ibm.websphere.simplicity.ShrinkHelper.DeployOptions.SERVER_ONLY;
+
+import org.jboss.shrinkwrap.api.ShrinkWrap;
+import org.jboss.shrinkwrap.api.spec.WebArchive;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.runner.RunWith;
+
+import com.ibm.websphere.simplicity.ShrinkHelper;
+
+import componenttest.annotation.Server;
+import componenttest.annotation.TestServlet;
+import componenttest.annotation.TestServlets;
+import componenttest.custom.junit.runner.FATRunner;
+import componenttest.topology.impl.LibertyServer;
+import componenttest.topology.utils.FATServletClient;
+import io.openliberty.microprofile.telemetry.internal_fat.apps.telemetry.BaggageServlet;
+import io.openliberty.microprofile.telemetry.internal_fat.apps.telemetry.ConfigServlet;
+import io.openliberty.microprofile.telemetry.internal_fat.apps.telemetry.MetricsDisabledServlet;
+import io.openliberty.microprofile.telemetry.internal_fat.apps.telemetry.OpenTelemetryBeanServlet;
+import io.openliberty.microprofile.telemetry.internal_fat.apps.telemetry.PatchTestApp;
+import io.openliberty.microprofile.telemetry.internal_fat.apps.telemetry.SpanCurrentServlet;
+import io.openliberty.microprofile.telemetry.internal_fat.apps.telemetry.Telemetry10Servlet;
+import io.openliberty.microprofile.telemetry.internal_fat.apps.telemetry.WithSpanServlet;
+
+@RunWith(FATRunner.class)
+public class TelemetryConfigEnvTest extends FATServletClient {
+
+    public static final String SERVER_NAME = "Telemetry10ConfigEnv";
+    public static final String APP_NAME = "TelemetryApp";
+
+    @Server(SERVER_NAME)
+    @TestServlets({
+                    @TestServlet(servlet = ConfigServlet.class, contextRoot = APP_NAME),
+    })
+    public static LibertyServer server;
+
+    @BeforeClass
+    public static void setUp() throws Exception {
+        WebArchive app = ShrinkWrap.create(WebArchive.class, APP_NAME + ".war")
+                        .addAsResource(ConfigServlet.class.getResource("microprofile-config.properties"), "META-INF/microprofile-config.properties")
+                        .addClasses(ConfigServlet.class);
+
+        ShrinkHelper.exportAppToServer(server, app, SERVER_ONLY);
+        server.addEnvVar("OTEL_SERVICE_NAME", "overrideDone");
+        server.addEnvVar("OTEL_SDK_DISABLED", "false");
+        server.startServer();
+    }
+
+    @AfterClass
+    public static void tearDown() throws Exception {
+        server.stopServer("CWNEN0047W.*ConfigServlet");
```

Comment:
What does this warning mean and why do we expect it? Please add a comment.

Looks like `CWNEN0047W: Resource annotations on the fields of the {0} class will be ignored. The annotations could not be obtained because of the exception : {1}` and I'm not sure why we'd expect that if the test is working correctly.